eg/servo-drive.js

```diff
@@ -39,8 +38,6 @@ board.on("ready", function() {
   wheels.both.cw();
 
   // Stop driving after 3 seconds
-  this.wait(3000, function() {
-    wheels.both.stop();
-  });
+  board.wait(3000, wheels.both.stop);
```

Comment:
This should be safe because `wait` does not bind `this`

Reply:
Ah, now it's clear. TY